vsts/vsts.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
           version: 2.1.x
           performMultiLevelLookup: true
           installationPath: $(Agent.ToolsDirectory)/dotnet
-      - powershell: ./build.ps1 -clean -configuration Release -build -run
+      - powershell: ./build.ps1 -clean -configuration Release -build -clear
```

Comment:
We'll still want to have a pipeline that runs all our samples. I think we should split out just the sample cleanup part to a different yaml and have that run independent of the entire sample run.

Once we move our samples out of this repo and into our sourcecode repo, we can merge the sample nightly run with our E2E nightly run.

Reply:
Agree. Eventually, I think we don't have this cleanup pipeline, test will clean up for its own test data and config.